src/profile.c

```diff
@@ -108,7 +108,7 @@ static double _recursiveProfilePrint(RedisModule_Reply *reply, ResultProcessor *
 
       case RP_SAFE_LOADER:
         printProfileType(RPTypeToString(rp->type));
-        printProfileGILTime(rp->GILTime);
+        printProfileGILTime(rs_wall_clock_convert_ns_to_ms_d(rp->rpGILTime));
```

Comment:
Regarding [previous comment](https://github.com/RediSearch/RediSearch/pull/7594/files#r2581403206)
This is the only place we access `rpGILTime`, and we know the `rp` type here.